### ISSUE TITLE
Official release

### ISSUE THREAD

**Issue by borissmidt** [jrapoport/grpc-web-devtools#3]
Could you officially release your fork as a plugin for firefox and chrome so people can use it without building it themself?

**Comment by jrapoport** [comment]
Yeah, it's been long enough. I'll look into this.

**Comment by borissmidt** [comment]
Is there any way i can help with this?

**Comment by jrapoport** [comment]
Not sure. I've never released an extension before. Was planning on digging in this weekend. Hopefully I can just post it and submit a link. 

Separately I'm wondering if we need to change the name to avoid confusion?

**Comment by borissmidt** [comment]
Btw after some reverse engineering with the original plugin i found out that i could do just push custom messages which is handy if you use the improbable-eng backend (it supports websockets).

But it seems that it doesn't work with big responses any idea how it could be fixed? 
https://github.com/SafetyCulture/grpc-web-devtools/issues/116

also it might be good to add it to the readme to make it easy for people to add support to other backends.

**Comment by stepan-romankov** [comment]
@[jrapoport](https://github.com/jrapoport) do you still have a plan to maintain this repo?

**Comment by jrapoport** [comment]
Yes. I've been busy working on a number of things and was planning to get caught up this week. For this repo there are a few open things I want to look at, including publishing the plugin to the Chrome. I'm not sure whether changing the name is more or less confusing wrt to the other repo. Any thoughts would be appreciated.

**Comment by stepan-romankov** [comment]
> Yes. I've been busy working on a number of things and was planning to get caught up this week. For this repo there are a few open things I want to look at, including publishing the plugin to the Chrome. I'm not sure whether changing the name is more or less confusing wrt to the other repo. Any thoughts would be appreciated.

Hi, thanks for reply, I'm asking cause we made a pull request and will be happy to see it in the release 
https://github.com/jrapoport/grpc-web-devtools/pull/7

**Comment by eeston** [comment]
@jrapoport I would really appreciate a FF release too! 👍

**Comment by jrapoport** [comment]
Hey guys, believe it or not I have been just completely slammed. But this is something I have been meaning to circle back on forever. 

I know there are some pending issues, but-- I'm gonna ignore them for now and focus on getting this out as an official release.

Once that's done, I can start to work through some of the open issues. 

I know this has been forever and a lot of you've been waiting for a really long time, but I'm committed to do it this week.

**Comment by jrapoport** [comment]
I just checked on the main repo though it looks like they did a release in 2023 that added some form of interceptor support. 

Is this fork actually still useful? I did a complete rewrite of all of their internal code (which frankly I think is a lot more solid than what they had the last time I looked).  But if you guys think that releasing/maintaining this isn't worth it after-all, let me know.

**Comment by VinceBT-BG** [comment]
Did you manage to make any of the GRPC chrome extensions work with their update ?

**Comment by jrapoport** [comment]
No, but I forked this originally to add / fix native interceptor support. 

It looks like they may have revived the project and [added that in 2023](https://github.com/SafetyCulture/grpc-web-devtools/pull/170). 

Or am I missing something?
